src/ingest/subscriptions/post.js

```diff
@@ -51,7 +75,7 @@ module.exports = async (req, res) => {
 			// delete datastore object
 			await datastore.delete('subscriptions', subscription.id);
 
-			// return error
+			// return 404 error
 			return res.sendStatus(404);
```

Comment:
should we include an error message about the topic?

Reply:
solved with https://github.com/swrlab/ard-eventhub/pull/26/commits/6b6e2f56d22464dba01fdd0b81374196161db8f4